dev/env.single-cluster-defaults

```diff
@@ -10,4 +10,7 @@ export CI_OCI_USERNAME=fleet-ci
 export CI_OCI_PASSWORD=foo
 export CI_OCI_CERTS_DIR=FleetCI-RootCA/
 
-export HELM_PATH=/usr/bin/helm
+# File with custom environment
+if [[ -f "$CUSTOM_ENV_FILE" ]]; then
+    source "$CUSTOM_ENV_FILE"
+fi
```

Comment:
Not sure this is necessary.

"Do it all" scripts are meant to source https://github.com/rancher/fleet/blob/main/dev/setup-cluster-config which is used to switch between 
`DEFAULT_CONFIG="dev/env.single-cluster-defaults"` and  `CUSTOM_CONFIG_FILE="env.single-cluster"`. Where `env.single-cluster` is already in `.gitignore`.


Well, `setup-cluster-config` seems a bit too much me, it's only called from two scripts. Those scripts could just source their default/custom config, duplicating the if switches might even help with readability.